Kaushik.akula/crosslink obs pipelines docs (#22785)
DataDog/documentation: content/en/logs/guide/best-practices-for-log-management.md

- [Alerting on indexed logs when the volume passes a specified threshold](#alert-when-an-indexed-log-volume-passes-a-specified-threshold)
- [Setting up exclusion filters on high-volume logs](#set-up-exclusion-filters-on-high-volume-logs)

> If you want to transform, redact sensitive data from, or more to your logs before they leave your environment, see how to [aggregate, process, and transform your logging data with Observability Pipelines][29].

Reviewer: ```suggestion
If you want to transform your logs or redact sensitive data in your logs before they leave your environment, see how to [aggregate, process, and transform your log data with Observability Pipelines][29].
```
